api/reader.graphql

```diff
@@ -87,13 +89,7 @@ type Voucher {
 "Top level queries"
 type Query {
   "Get input based on its identifier"
-  input(index: Int!): Input!
-  "Get a voucher based on its index"
-  voucher(voucherIndex: Int!, inputIndex: Int!): Voucher!
```

Comment:
> CI failed with error `level=error msg="thresholds on metrics 'checks' have been crossed"` Not sure what's the error

@ZzzzHui k6 and loadtest try queries on this "endpoints". Maybe we need revert schema this guys.

Reply:
is that an automatic test? is it possible to remove the tests for those queries that we don't need?

Reply:
My recommendation is to keep other queries and increment other.
Because removing them can be a mess.

Reply:
could you elaborate a bit more please.
What's "increment other"
and why removing them is a mess. It's a mess only for the CI or for other things?

Reply:
@sandhilt ⬆️

Reply:
Check this PR: #225